modules/ROOT/pages/endpoints.adoc

```diff
@@ -0,0 +1,73 @@
+= Endpoints
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+An _endpoint_ is the entry point to a service, a process, or a queue or topic destination in service-oriented architecture. In APM, an endpoint defines the protocol, address, and other details specific to an exchange of messages between partners.
+
+== Endpoint Name, Protocol and Type
+
+[%header,cols="3s,7a"]
+|===
+|Setting |Description
+
+|Name
+
+|Name for this endpoint
+
+Required:: Yes
+
+Default:: Auto-populates based on the following values:
+
+* *Protocol*
+* *Type*
+
+
+|Protocol
+
+|Protocol for this endpoint
+
+Required?::
+No
+
+Valid values::
+
+* 'AS2'
```

Comment:
Why are these values between `'` characters? I think this could be an incorrect formatting, the proper character is ``` to convert 'AS2' to `AS2`. Please confirm if that's the case and replace where necessary.